dask/dataframe/groupby.py

```diff
@@ -1881,7 +1888,9 @@ def get_group(self, key):
         )
 
     @_aggregate_docstring()
```

Comment:
This is looking good @ChrisJar - Thanks!

Can we update [_aggregate_docstring](https://github.com/dask/dask/blob/0532bb39094e8ab42525f65926f5aab773f51fbc/dask/dataframe/groupby.py#L1154) to add a description for `**kwargs` (and to clarify that these additional key-word arguments will be interpreted as named aggregations)?